README.md

```diff
@@ -33,70 +34,77 @@ This table contains all the `WinetProxy` instances that act as entry-points for
   | ------------ | --------- | ---------------------
 
 
-## WitnetRequest
+## WitnetRequestBase
 
- A `WitnetRequest` is constructed around a `bytes` value containing a well-formed Witnet RADON script data request serialized
- using Protocol Buffers. The `WitnetRequest` base contract provides one single method:
+Used as a means to encapsulate Witnet Data Requests, that can eventually be posted to a `WitnetRequestBoard` implementation. The `WitnetRequestBase` is constructed from the CBOR-encoded serialization of a [Witnet Data Request](https://docs.witnet.io/protocol/data-requests/overview/), implementing the `IWitnetRequest` interface.
+
+### IWitnetRequest
+
+ The `IWitnetRequest` interface defines two methods:
 
 - **`bytecode()`**:
   - _Description_:
-    - Returns Witnet RADON script as a CBOR-encoded `bytes`. 
+    - Returns Witnet Data Request as a CBOR-encoded `bytes`. 
+
+- **`codehash()`**:
+  - _Description_:
+    - Returns Witnet Data Request as a CBOR-encoded `bytes`. 
```

Comment:
This should say "Returns SHA256 hash of Witnet Data Request as a CBOR-encoded `bytes`"